### DISCUSSION TITLE
Packaging Python Dependencies on GitHub: Dependency Conflict Resolution

### DISCUSSION THREAD

**Discussion by Dorothy-Reyes** [pybamm-team/PyBaMM#3350] [Q&A]
Hi,
I've encountered a frustrating dependency conflict issue while packaging my Python project on GitHub. How can I efficiently resolve conflicting dependencies to ensure a smooth packaging process? Seeking expert advice from the GitHub community!

**Comment by brosaplanella** [comment]
See https://github.com/pybamm-team/PyBaMM/discussions/3353#discussioncomment-7032538
